object-cache.php

```diff
@@ -3,7 +3,7 @@
 /*
 Plugin Name: Memcached
 Description: Memcached backend for the WP Object Cache.
-Version: 3.0.1
+Version: 3.1.0
```

Comment:
Please see https://github.com/Automattic/wp-memcached/pull/45 I think we should merge this first then bump this to `3.2.0`. Master currently has what I would consider a 'minor' change as per semver which has already been merged into master.

Reply:
I was thinking of merging this first actually: https://github.com/Automattic/wp-memcached/pull/52

As the trunk on wp.org and wp-memcached GH repos are totally out of sync.

But I agree. The other one should probably be 3.1.0, and this 3.2.0.

Reply:
yeah agreed #52 prob more important, find it a bit odd they have diverged

Reply:
Me too. But will get them in sync soon. Thanks for checking.